lib/constable_web/templates/announcement/index.html.eex

```diff
@@ -27,7 +27,7 @@
 
     <%= if !@show_all do %>
       <%= link to: Routes.interest_path(@conn, :index),
-        data: [role: "view-all-interests"],
+        data: [role: "view-all-interests", turbolinks: false],
```

Comment:
Note for reviewers: this is where we ran into issues with Turbolinks navigating to the interests page: it does not load the LiveView JS when that happens. We could work around that but for now this is fine.